robocode.installer/src/chocolatey/robocode.nuspec

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>robocode</id>
+    <version>1.1.1.1</version>

```

Comment:
I don't remember if this is somehow auto-modified or needs attention

Reply:
@pavelsavara Thank you for paying attention to this detail. :)
I have just verified the `gradlew chocoBuild` and checked that the version of `robocode.nuspec` inside the .nupkg file gets the correct version. So currently 1.1.1.1 is being replaced with 1.9.4.2.